src/routes/automations.ts

```diff
@@ -0,0 +1,34 @@
+import { FastifyInstance } from 'fastify';
+import { Automation, automations } from '../integrations/automation';
+import { HttpError } from '../integrations/retry';
+
+export default async function (fastify: FastifyInstance): Promise<void> {
+  const counter = fastify.meter?.createCounter('automations', {
+    description: 'How many automations were triggered',
+  });
+
+  fastify.post<{ Body: Record<string, unknown>; Params: { name: Automation } }>(
+    '/:name',
+    async (req, res) => {
+      const { body, userId } = req;
+      const name = req.params.name;
+      const auto = automations[name];
+      if (!auto) {
+        return res.status(404).send();
+      }
+      if (!userId) {
+        return res.status(401).send();
+      }
```

Comment:
Might be blind at this point, but what prevents anyone for just supplying their userId? Should this be limited only to team members?

Reply:
`userId` is extracted from the JWT so you can't trick it :)

Reply:
And our JWT knows to limit this endpoint just for team members or this supposed to be "open" to any registered user?

Reply:
Should be opened to any registered user. So the idea is to offload some of the work to automation services. Since we currently only have one in place I wasn't 100% how to generalize. Maybe once we get the second one, we'll see if all of them should be public or just a subset. Anyway we can fix this endpoint according to future requirements. For now we have only one

Reply:
Ok, then its good, I was not sure if even this one should be open. LGTM!